math/genvector/inc/Math/GenVector/eta.h

```diff
@@ -52,8 +52,10 @@ namespace ROOT {
               static const Scalar big_z_scaled = pow(std::numeric_limits<Scalar>::epsilon(), static_cast<Scalar>(-.25));
 
               Scalar z_scaled = z/rho;
+              using std::fabs;
               if (std::fabs(z_scaled) < big_z_scaled) {
-                 return log(z_scaled + sqrt(z_scaled * z_scaled + 1.0));
+                 using std::sqrt;
+                 return log(z_scaled + std::sqrt(z_scaled * z_scaled + 1.0));
```

Comment:
...and same for all occurrences of `log`. Sorry - I seem to find them only one by one :-/

Reply:
No problem. I'll change it :)